lib/active_admin/views/components/status_tag.rb

```diff
@@ -50,7 +46,12 @@ def build(*args)
         super(content, options)
 
         add_class(status_to_class(status)) if status
-        add_class(type.to_s) if type
+
+        if type
+          Deprecation.warn "the `type` parameter is deprecated! Provide as a class: `status_tag(status, class: 'abc')`."
```

Comment:
I think we can still improve the message a bit. What about this:

The `type` parameter has been deprecated. Provide the "#{type}" type as a class option instead. For example, `status_tag(status, :ok, class: "abc")` would change to `status_tag(status, class: "ok abc")`. Also note that the #{type} CSS class will be removed too, so you'll have to provide the CSS rules yourself. See https://github.com/activeadmin/activeadmin/blob/master/CHANGELOG.md#110- for more information.